samples/BasicXrApp/OpenXrProgram.cpp

```diff
@@ -631,7 +642,7 @@ namespace {
         bool m_sessionRunning{false};
         XrSessionState m_sessionState{XR_SESSION_STATE_UNKNOWN};
         XrEnvironmentBlendMode m_environmentBlendMode{};
-    };
+    }; // namespace
```

Comment:
This comment is wrong - that brace aligns with `struct OpenXrProgram : IOpenXrProgram {`.